kate/Cargo.toml

```diff
@@ -12,7 +12,7 @@ license = "Apache-2.0"
 poly-multiproof = { git = "https://github.com/availproject/poly-multiproof", default-features = false, tag = "v0.0.1", optional = true }
 
 # Internal
-avail-core = { path = "../core", default-features = false, feature = "runtime" }
+avail-core = { path = "../core", default-features = false }
```

Comment:
I am not sure if this change will be an issue regarding LC and/or node built.
Have you tested those builds?

Reply:
@fmiguelgarcia Hey, the node and LC compiles correctly using this branch. Also it seems like kate was using avail-core with this feature enabled but it was actually not used and produced a warning.